homeassistant/components/mawaqit/translations/ar.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
Remove all translations files. Only `strings.json` is needed as we handle the translations via Lokalize.
See https://developers.home-assistant.io/docs/internationalization/core

Reply:
Done !